qa/suites/orch/cephadm/workunits/task/test_rgw_multisite.yaml

```diff
@@ -0,0 +1,39 @@
+roles:
+- - host.a
+  - mon.a
+  - mgr.a
+  - osd.0
+- - host.b
+  - mon.b
+  - mgr.b
+  - osd.1
+- - host.c
+  - mon.c
+  - osd.2
+tasks:
+- install:
+- cephadm:
+- cephadm.shell:
+    host.a:
+      - ceph mgr module enable rgw
+- rgw_module.apply:
+    specs:
+      - rgw_realm: myrealm1
+        rgw_zonegroup: myzonegroup1
+        rgw_zone: myzone1
+        spec:
+          rgw_frontend_port: 5500
+- cephadm.shell:
+    host.a:
+      - |
+        set -e
+        set -x
+        TOKENS=$(ceph rgw realm tokens)
+        echo $TOKENS | jq --exit-status '.[0].realm == "myrealm1"'
+        echo $TOKENS | jq --exit-status '.[0].token'
+        TOKEN_JSON=$(ceph rgw realm tokens | jq -r '.[0].token' | base64 --decode)
```

Comment:
Test doesn't seem to quite work yet. I think it's failing here
```
2023-03-22T15:14:28.729 INFO:teuthology.orchestra.run.smithi019.stderr:+ TOKENS='[
2023-03-22T15:14:28.729 INFO:teuthology.orchestra.run.smithi019.stderr:    {
2023-03-22T15:14:28.729 INFO:teuthology.orchestra.run.smithi019.stderr:        "realm": "myrealm1",
2023-03-22T15:14:28.729 INFO:teuthology.orchestra.run.smithi019.stderr:        "token": "master zone has no endpoint"
2023-03-22T15:14:28.730 INFO:teuthology.orchestra.run.smithi019.stderr:    }
2023-03-22T15:14:28.730 INFO:teuthology.orchestra.run.smithi019.stderr:]'
2023-03-22T15:14:28.730 INFO:teuthology.orchestra.run.smithi019.stderr:+ echo '[' '{' '"realm":' '"myrealm1",' '"token":' '"master' zone has no 'endpoint"' '}' ']'
2023-03-22T15:14:28.730 INFO:teuthology.orchestra.run.smithi019.stderr:+ jq --exit-status '.[0].realm == "myrealm1"'
2023-03-22T15:14:28.760 INFO:teuthology.orchestra.run.smithi019.stdout:true
2023-03-22T15:14:28.761 INFO:teuthology.orchestra.run.smithi019.stderr:+ echo '[' '{' '"realm":' '"myrealm1",' '"token":' '"master' zone has no 'endpoint"' '}' ']'
2023-03-22T15:14:28.761 INFO:teuthology.orchestra.run.smithi019.stderr:+ jq --exit-status '.[0].token'
2023-03-22T15:14:28.791 INFO:teuthology.orchestra.run.smithi019.stdout:"master zone has no endpoint"
2023-03-22T15:14:28.792 INFO:teuthology.orchestra.run.smithi019.stderr:++ ceph rgw realm tokens
2023-03-22T15:14:28.792 INFO:teuthology.orchestra.run.smithi019.stderr:++ jq -r '.[0].token'
2023-03-22T15:14:28.792 INFO:teuthology.orchestra.run.smithi019.stderr:++ base64 --decode
2023-03-22T15:14:29.424 INFO:journalctl@ceph.mon.c.smithi202.stdout:Mar 22 15:14:29 smithi202 ceph-mon[96864]: Deploying daemon rgw.myrealm1.myzone1.smithi202.bswwyw on smithi202
2023-03-22T15:14:29.425 INFO:journalctl@ceph.mon.c.smithi202.stdout:Mar 22 15:14:29 smithi202 ceph-mon[96864]: from='client.14730 -' entity='client.admin' cmd=[{"prefix": "rgw realm tokens", "target": ["mon-mgr", ""]}]: dispatch
2023-03-22T15:14:29.429 INFO:teuthology.orchestra.run.smithi019.stderr:base64: invalid input
2023-03-22T15:14:29.430 INFO:teuthology.orchestra.run.smithi019.stderr:+ TOKEN_JSON=$'\231\253-z'
2023-03-22T15:14:29.445 INFO:journalctl@ceph.mon.a.smithi019.stdout:Mar 22 15:14:29 smithi019 ceph-mon[94540]: Deploying daemon rgw.myrealm1.myzone1.smithi202.bswwyw on smithi202
2023-03-22T15:14:29.446 INFO:journalctl@ceph.mon.a.smithi019.stdout:Mar 22 15:14:29 smithi019 ceph-mon[94540]: from='client.14730 -' entity='client.admin' cmd=[{"prefix": "rgw realm tokens", "target": ["mon-mgr", ""]}]: dispatch
2023-03-22T15:14:29.621 INFO:journalctl@ceph.mon.b.smithi090.stdout:Mar 22 15:14:29 smithi090 ceph-mon[96538]: Deploying daemon rgw.myrealm1.myzone1.smithi202.bswwyw on smithi202
2023-03-22T15:14:29.621 INFO:journalctl@ceph.mon.b.smithi090.stdout:Mar 22 15:14:29 smithi090 ceph-mon[96538]: from='client.14730 -' entity='client.admin' cmd=[{"prefix": "rgw realm tokens", "target": ["mon-mgr", ""]}]: dispatch
2023-03-22T15:14:30.135 DEBUG:teuthology.orchestra.run:got remote process result: 1
```
https://pulpito.ceph.com/adking-2023-03-15_01:34:51-orch:cephadm-wip-adk-testing-2023-03-14-1905-distro-default-smithi/7208451

Reply:
@adk3798 it seems that when the checks are ran, the rgw daemons are not created yet. Not sure if we should add some active-wait or in fact the rgw are not being created for some other reason.